check-osp/utils.py

```diff
@@ -28,3 +28,12 @@ def ssh_client(hostname, username, password):
         # authenticates, and then creates stream tunnels, called channels, across the session.
         # If connection is not established, returns None.
         raise timeout("No pingable IP found")
+
+
+def get_services_status_list(ssh):
+    """
+    This function fetches list of all units of type services from given host and returns in dict.
+    """
+    stdin, stdout, stderr = ssh.exec_command("systemctl list-units --type service --state=running"
```

Comment:
my concern here is it checks running state but I see the check is designed to handle check of different states. Is it possible to adapt this function for more general purpose as well?

Reply:
Angelina, we are not checking things other than  "running/exited" so I did add only those too. I can make it generic but wasn't sure if that's needed. Let me still check if I can quickly add it.

Reply:
Removed filtering, listing all service units now.